### ISSUE TITLE
Doesn't work

### ISSUE THREAD

**Issue by corysimmons** [pedronauck/docz#462]
- `npx create-react-app docz-test`
- `cd docz-test`
- `yarn add -D docz`
- create these 2 files in `/src`

```js
// Button.js

import React from 'react'

const Button = () => <button>Click me</button>

export default Button
```

```
// Button.mdx

---
name: Button
menu: Components
---

import { Playground } from 'docz'
import Button from './Button'

# Button

## Basic Usage

<Playground>
  <Button />
</Playground>
```

- `yarn docz dev`

![image](https://user-images.githubusercontent.com/224636/48816394-ffff1680-ed0f-11e8-97de-a9adab8fa347.png)

I would expect the default index page to be something other than 404.

![image](https://user-images.githubusercontent.com/224636/48816410-0f7e5f80-ed10-11e8-8f59-6a6e8b60de29.png)

No idea why this doesn't work. I've tried creating `/src/components` as well.

If I'm doing something wrong, please write easier to follow "Getting Started" docs with a bare bones CRA example.

**Comment by praneybehl** [comment]
Yeah same!

**Comment by KingScooty** [comment]
I'm getting the same error whenever I use the <Playground /> component.

**Comment by yyynnn** [comment]
yarn 1.10
node 10

same here, basic setup with ts, empty mdx with just text -> page not found / no crashes

**Comment by pedronauck** [comment]
Fixed on [v0.12.13](https://github.com/pedronauck/docz/releases/tag/v0.12.13) ✅
